src/index.js

```diff
@@ -1,11 +1,17 @@
 ( function( wp ) {
     const registerPlugin = wp.plugins.registerPlugin;
     const createElement = wp.element.createElement;
-    const { Modal, Button } = wp.components;
+    const { Modal, Button, RadioControl } = wp.components;
     const { withState } = wp.compose;
     
-    const insertTemplate = () => {
-        fetch('https://www.mocky.io/v2/5cd3fb89350000de307a5211')
+    const insertTemplate = template => {
+        console.log( 'Using template', template );
```

Comment:
Do you want to remove this?

Side note: we need `eslint` configured to catch this stuff like in Core.